src/index.d.ts

```diff
@@ -4616,7 +4616,14 @@ declare namespace admin.messaging {
     silent?: boolean;
 
     /**
-     * An identifying tag for the notification.
+     * Identifier used to replace existing notifications in the notification drawer.
+     *
+     * If not specified, each request creates a new notification.
+     *
+     * If specified and a notification with the same tag is already being shown,
+     * the new notification replaces the existing one in the notification drawer.
+     *
+     * **Platforms:** Android
```

Comment:
This is part of the `WebpushNotification` interface. The `tag` you're trying to document is in the `NotificationMessagePayload` interface (line 4662).

Reply:
Yikes!   Sorry about that confusion.

I think I have everything back in order now, and staged both pages to make sure (legacy payload and WebPushNotification.html).  Thanks!